apis/visibility/v1alpha1/types.go

```diff
@@ -27,6 +27,11 @@ import (
 type PendingWorkload struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Priority               int32  `json:"priority"`
+	LocalQueueName         string `json:"localqueuename"`
+	PositionInClusterQueue int32  `json:"positioninclusterqueue"`
+	PositionInLocalQueue   int32  `json:"positioninlocalqueue"`
```

Comment:
```suggestion
	Priority               int32  `json:"priority"`
	LocalQueueName         string `json:"localQueueName"`
	PositionInClusterQueue int32  `json:"positionInClusterQueue"`
	PositionInLocalQueue   int32  `json:"positionInLocalQueue"`
```

Can we use camel case as well as other APIs?